projects/igniteui-angular/src/lib/query-builder/query-builder.component.html

```diff
@@ -391,8 +391,8 @@ <h6 class="igx-filter-empty__title">
             >
             </ng-container>
             <button type="button"
-                igxButton="icon"
-                [displayDensity]="this.displayDensity"
+                igxIconButton="flat"
+                [style.--component-size]="getComponentSizeStyles()"
```

Comment:
Just like my other comment, can we set the size in the theme itself? I believe this will require a bit more work, but it should be doable.